views/forms/ChartForm.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react'
+import Button from '@material-ui/core/Button'
+
+import { createChart } from '../fe-utils/fetchUtil'
+
+import Form from './Form'
+import BarChartFields from './BarChartFields'
+
+const ChartForm = ({ classes }) => {
+  const [formValues, setFormValues] = useState({})
```

Comment:
What do you think about defaulting the form values?

```suggestion
  const [formValues, setFormValues] = useState({
    title: '',
    assessment: '',
    variable: '',
    fieldLabelValueMap: [],
  })
```

Reply:
I think that right now the other types of charts and the values to generate them are still unknown. A title property seems like the most generic across the board so something like this could work.
```suggestion
  const [formValues, setFormValues] = useState({ title: '' })
```

Reply:
they're unknown, but we'll have to default to something 🤷

Reply:
[78b1245](https://github.com/AMP-SCZ/dpdash/pull/84/commits/78b12458361f85aeb0919c885be4ca8c8f6c047b)